mysql-test/suite/sys_vars/t/binlog_ignore_db.test

```diff
@@ -0,0 +1,15 @@
+--echo #
+--echo # MDEV-28672 Show "--binlog_ignore_db" state in SYSTEM VARIABLES
+--echo #
+
+# Confirm that "--binlog_ignore_db" option has the default value.
+SELECT @@binlog_ignore_db AS EXPECT_NULL;
+
+# `READ ONLY` variables must not be able to be reset.
+--Error ER_INCORRECT_GLOBAL_LOCAL_VAR (1238): Variable 'binlog_do_db' is a read only variable
+SET @@GLOBAL.binlog_ignore_db=1;
+
+# Restart the server with "--binlog_ignore_db='1'" option
+--let $restart_parameters = "--binlog_ignore_db='1'"
+--source include/restart_mysqld.inc
+SELECT @@binlog_ignore_db AS EXPECT_1;
```

Comment:
For `binlog_do_db` and `binlog_ignore_db` tests, can you add another test case to ensure multiple databases are composed into a single comma separated list correctly?

Reply:
Thanks @bnestere ! That's a good idea! (Actually I tested this case manually before but probably should also add it into the test file)